src/routes/products.js

```diff
@@ -1,30 +1,54 @@
 const router = require('express').Router();
+const { PORT } = require('../configs/constants');
 
 const ProductModel = require('../../models/Products');
 
-router.get('/', async (_, res, next) => {
+router.get('/:page', async (req, res, next) => {
+  const perPage = 25;
+  const page = req.params.page || 1;
+
   try {
-    const result = await ProductModel.find({});
+    const result = await ProductModel.find({})
+      .skip(perPage * page - perPage)
+      .limit(perPage);
+
+    const nextPage =
+      result.length < perPage
+        ? 'this is the last page'
+        : `http://localhost:${PORT}/api/products/${Number(page) + 1}`;
```

Comment:
I really prefer returning `null` instead of a custom string like `'this is the last page'` so the code in the frontend can easily handle the requests. This is an added complexity to how urls are validated on the client side.

Additionally, how about not returning the `http://localhost:${PORT}` so the client has to build the full url? This way the server is agnostic about what environment it's currently running at and the client can choose where to point.

Reply:
?page=1

Reply:
ready, ajustado en nextpage, cambie los req.params por req.query tambien  y por ultimo he ajustado el json de postman con las url correctas.